packages/blade/src/components/Carousel/Carousel.web.tsx

```diff
@@ -400,7 +402,7 @@ const Carousel = ({
 
   const carouselContext = React.useMemo<CarouselContextProps>(() => {
     return {
-      visibleItems,
+      visibleItems: _visibleItems,
```

Comment:
fixes a bug where when you add `<Carousel visibleItems={2} />` and open the component on mobile device it still stays as visibleItems, but instead on mobile we should automatically change to use 1 visibleItems.